tests/e2e/utils/kind.go

```diff
@@ -163,102 +163,46 @@ func DeleteCluster(name string) error {
 }
 
 // StartClusterLink creates a cluster, and launches clusterlink
-func StartClusterLink(name, cPortLocal, manifests string, cPort uint16, cpType string) error {
-	certs := "./mtls"
+func StartClusterLink(name, cPortLocal string, cPort uint16) error {
 	cPortStr := strconv.Itoa(int(cPort))
-	clusterIP, err := createCluster(name)
+	_, err := createCluster(name)
 	if err != nil {
 		return err
 	}
 
-	if cpType == "new" {
-		clAdm := ProjDir + "/bin/cl-adm "
-		err := runCmdInDir(clAdm+" create peer --name "+name, testOutputFolder)
-		if err != nil {
-			return err
-		}
-
-		err = runCmd("kind load docker-image cl-controlplane cl-dataplane cl-go-dataplane --name=" + name)
-		if err != nil {
-			return err
-		}
-
-		err = runCmd("kubectl apply -f " + testOutputFolder + name + "/k8s.yaml")
-		if err != nil {
-			return err
-		}
-
-		err = IsPodReady("cl-controlplane")
-		if err != nil {
-			return err
-		}
-
-		err = IsPodReady("cl-dataplane")
-		if err != nil {
-			return err
-		}
-		err = runCmd("kubectl delete service cl-dataplane")
-		if err != nil {
-			return err
-		}
-
-		err = runCmd("kubectl create service nodeport cl-dataplane --tcp=" + cPortLocal + ":" + cPortLocal + " --node-port=" + cPortStr)
-		if err != nil {
-			return err
-		}
-
-	} else {
-		err = runCmd("kind load docker-image mbg --name=" + name)
-		if err != nil {
-			return err
-		}
-
-		err = runCmd("kubectl apply -f " + manifests + "mbg-role.yaml")
-		if err != nil {
-			return err
-		}
-
-		err = runCmd("kubectl create -f " + manifests + "mbg.yaml")
-		if err != nil {
-			return err
-		}
-
-		err = runCmd("kubectl create -f " + manifests + "dataplane.yaml")
-		if err != nil {
-			return err
-		}
-
-		err = IsPodReady("mbg")
-		if err != nil {
-			return err
-		}
+	clAdm := ProjDir + "/bin/cl-adm "
+	err = runCmdInDir(clAdm+" create peer --name "+name, testOutputFolder)
+	if err != nil {
+		return err
+	}
 
-		gwPod, _ := GetPodNameIP("mbg")
-		err = IsPodReady("dataplane")
-		if err != nil {
-			return err
-		}
+	err = runCmd("kind load docker-image cl-controlplane cl-dataplane cl-go-dataplane --name=" + name)
+	if err != nil {
+		return err
+	}
 
-		err = runCmd("kubectl create service nodeport dataplane --tcp=" + cPortLocal + ":" + cPortLocal + " --node-port=" + cPortStr)
-		if err != nil {
-			return err
-		}
+	err = runCmd("kubectl apply -f " + testOutputFolder + name + "/k8s.yaml")
```

Comment:
ditto - can/should we use `client-go` directly? It can work with YAMLs as well

Reply:
You are right, but most of this code will be replaced with the new framework so I will not fix it, I just wanted to start the cleanup and remove the code of the old controlplane.